platform-emf/src/main/java/com/softicar/platform/emf/attribute/IEmfAttribute.java

```diff
@@ -227,6 +228,14 @@
 	 */
 	IDomElement createTabularDisplay(R tableRow);
 
+	/**
+	 * Returns a factory of display elements that contain help text for this
+	 * {@link IEmfAttribute}.
```

Comment:
sounds complicated and too specific, how about
```suggestion
	 * Returns an optional factory for a display that describes this
	 * {@link IEmfAttribute}.
```

Reply:
did something similar